specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/FailoverDatabases.json

```diff
@@ -37,6 +37,21 @@
             "required": true,
             "type": "string"
           },
+          {
```

Comment:
Please, append new parameters to the end of the parameter array.

Reply:
This is a generated file so we can't move this to the end of the parameter array